### PR TITLE
feat(pci.kube.reset): add proxy mode

### PR DESCRIPTION
**Pull request by y4nnL**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/k8s-proxy-mode`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #TAPC-87
| License          | BSD 3-Clause

- [X] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [X] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by qpavy — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset/reset.html [hunk 1]
```diff
@@ -1,77 +1,173 @@
...
-            data-network="$ctrl.model.network"
-        ></gateway-management-component>
-    </oui-modal>
+    <div class="form-group">
+        <oui-button
+            class="mr-3"
+            data-type="button"
+            data-variant="secondary"
+            data-size="l"
+            data-on-click="-$ctrl.onResetCancel()"
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset/reset.html:156]: Is it by mistake "-"?

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset/reset.html [hunk 2]
```diff
@@ -1,77 +1,173 @@
...
+                    <span
+                        class="fa fa-question-circle"
+                        data-oui-tooltip="{{:: ('pci_projects_project_kubernetes_service_reset_mode_' + key) | translate }}"
+                        data-oui-tooltip-placement="right"
+                        aria-hidden="true"
+                    ></span>
+                </td>
+                <td class="oui-table__cell">
+                    <span
+                        data-ng-if="value === 'PT0S'"
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset/reset.html:87]: 'PT0S' can be taken from constant file

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset/reset.html [hunk 3]
```diff
@@ -1,77 +1,173 @@
...
+                        data-translate="pci_projects_project_kubernetes_service_reset_mode_duration_zero"
+                    ></span>
+                    <span
+                        data-ng-if="value === 'PT1S'"
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset/reset.html:91]: same here

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset/reset.html (hunk 1 above)
Comment (y4nnL) [reply packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset/reset.html:156]: Nice catch :+1:

**Review by sachinrameshn — APPROVED** [review]

**Pull request merged**